internal/app/plan.go

```diff
@@ -86,25 +86,57 @@ func (p *plan) exec() {
 		log.Info("Nothing to execute")
 	}
 
+	wg := sync.WaitGroup{}
+	sem := make(chan struct{}, flags.parallel)
+	var fail bool
+	var priorities []int
+	pl := make(map[int][]orderedCommand)
 	for _, cmd := range p.Commands {
-		log.Notice(cmd.Command.Description)
-		result := cmd.Command.exec()
-		if cmd.targetRelease != nil && !flags.dryRun && !flags.destroy {
-			cmd.targetRelease.label()
+		priorities = append(priorities, cmd.Priority)
```

Comment:
this can have more than one priority of the same value causing it to run more than once for specific charts. priorities array must contain unique values before it goes to loop in 99 line

Reply:
good catch, I'll update the PR

Reply:
done, the list of priorities is now deduplicated.